src/vehicle/models/pod_data.go

```diff
@@ -64,17 +66,44 @@ func getMeasurements(values []excelAdapterModels.Value) map[string]Measurement {
 	return measurements
 }
 
-func parseRange(literal string) [2]string {
+func parseRange(literal string) []*float64 {
 	if literal == "" {
-		return [2]string{"", ""}
+		return make([]*float64, 0)
 	}
 
-	split := strings.Split(strings.TrimSuffix(strings.TrimPrefix(literal, "["), "]"), ",")
-	if len(split) != 2 {
+	strRange := strings.Split(strings.TrimSuffix(strings.TrimPrefix(strings.Replace(literal, " ", "", -1), "["), "]"), ",")
+
+	if len(strRange) != 2 {
 		log.Fatalf("pod data: parseRange: invalid range %s\n", literal)
 	}
 
-	return [2]string{split[0], split[1]}
+	numRange := make([]*float64, 0)
+
+	if strRange[0] != "" {
```

Comment:
it can be refactored a lot by extracting to methods and using guard clauses

Reply:
I just moved this file, I leave those changes for the next PR